CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Build
 
  * Add smoke tests for the binary server. See [#1075](https://github.com/redhat-developer/vscode-xml/issues/1075).
+ * Do not build Alpine Linux-specific version. See [#1083](https://github.com/redhat-developer/vscode-xml/pull/1083)
```

Comment:
Is it worth saying "Drop support for Alpine Linux-specific version." ?

Reply:
yeah